posts/flat-file-exports/flat-files-compare.qmd

```diff
@@ -0,0 +1,611 @@
+| x_global_px |float | The x position (in pixels) relative to the tissue.  | 
+| y_global_px |float | The y position (in pixels) relative to the tissue. | 
+| x_local_px |float | The x position (in pixels) relative to the given FOV. | 
+| y_local_px | float |The y position (in pixels) relative to the given FOV. | 
+| z | Int | The z plane. | 
+| target | string | The name of the target. | 
+| CellComp&#x2021; | string | Subcellular location of target. | 
+
+: Description of the legacy transcripts file format. &#x2021; = contents changed between versions {#tbl-tx-leg tbl-colwidths="[10, 10, 80]"}
+
+
+Example:
+```{r}
+#| eval: false
+#| echo: false
+
+tx_file_old <- read.csv("path/to/NSCLC/Lung5_Rep1/Lung5_Rep1-Flat_files_and_images/Lung5_Rep1_tx_file.csv", header=TRUE)
+tx_file_old %>% head(10)
+```
+
+
+```
+  fov cell_ID x_global_px y_global_px x_local_px y_local_px  z  target CellComp
+  1       0    6757.402    158836.4   3568.513  3619.7375 11   NEAT1        0
+  1       0    5111.389    156060.2   1922.500   843.5334 11   NEAT1        0
+  1       0    7860.461    157809.3   4671.572  2592.6715 11    CCR2        0
+  1       0    3790.489    155553.9    601.600   337.2168 11 HLA-DRA        0
+  1       0    3290.639    158023.6    101.750  2806.9750 11 HLA-DRA Membrane
+  1       0    7020.160    158656.3   3831.271  3439.6000 11     VHL        0
+  1       0    4252.914    157003.0   1064.025  1786.3376 11    FZD5  Nuclear
+  1       0    5987.309    157572.5   2798.420  2355.8000 11    CD37        0
+  1       0    5586.849    157774.2   2397.960  2557.5599 11   ATG12 Membrane
+```
+
+## AtoMx SIP v1.3.2
+
+| **Column Name** 	| **Type** 	| **Description** 	|
+|----------	|----------	|-----------------	|
+| fov                | Int | The field of view (FOV) number.  |
+| cell_ID            | Int | Unique identifier for a single cell within a given FOV. To make a unique identifier for a cell within the whole sample use both the `fov` and `cell_ID` columns. |
+| cell$^{*}$                 | string | A study-wide unique cell identifier. Combination of c(ell), slide ID, and `cell_ID`. Note that this is equivalent to `cell_ID` in `napari-cosmx`.  |
+| x_local_px |float | The x position (in pixels) relative to the given FOV. | 
+| y_local_px | float |The y position (in pixels) relative to the given FOV. | 
+| x_global_px |float | The x position (in pixels) relative to the tissue.  | 
+| y_global_px |float | The y position (in pixels) relative to the tissue. | 
+| z | Int | The z plane. | 
+| target | string | The name of the target. | 
+| CellComp&#x2021; | string | Subcellular location of target. | 
+
+: Description of the baseline transcripts file format for AtoMx SIP v1.3.2. Δ = column name change between versions; * = new column in 1.3.2 compared to legacy {#tbl-tx-132 tbl-colwidths="[10, 10, 80]"}
```

Comment:
Add symbol for contents changed. I also don't see any column name change symbols - can this be removed from the legend?

Reply:
done.